_docs/concepts/what-is-istio/goals.md

```diff
@@ -8,7 +8,9 @@ layout: docs
 type: markdown
 ---
 
-This page the core principles that guide Istio's design.
+This page outlines the core principles that guide Istio's design and explains some key design decisions based on these goals.
```

Comment:
Repeats 'design', maybe just remove the second one?

In the other file, you changed "page" to "document". Do we want that here too? Note the whole web site uses "page" at this point...